### ISSUE TITLE
Fill CoL gap: class/order **Diplura**

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#577]
_For consideration of Taxonomy Group (@dhobern & @olafbanki), 2023-11-02:_ 

**Dataset title**
Global checklist published in the paper of 2021: **Diversity, ecology, distribution and biogeography of Diplura** by Alberto Sendra, Alberto Jiménez-Valverde, Jesús Selfa, Ana Sofia P. S. Reboleira
https://resjournals.onlinelibrary.wiley.com/doi/10.1111/icad.12480

**Dataset contact & access**
The checklist is available as a spreadsheet as a supplement to the paper
https://resjournals.onlinelibrary.wiley.com/doi/10.1111/icad.12480
Also, it might be available with extended data or as a databse from the authors.
@gdower may crawl data in the CLB.

**Taxonomic group & CoL sector**
It's gap in the CoL: kingdom: Animalia > phylum: Arthropoda > class: Diplura 

**Dataset description**
Complete checklist with 1008 accepted species, which belong to 141 genera and 10 families

**Comment by yroskov** [comment]
@dhobern, as soon as TG give a green light, I'll contact authors for more information about the checklist and permission to use data in the CoL

**Comment by mdoering** [comment]
I have converted the excel sheet to coldp here:
https://github.com/CatalogueOfLife/data-diplura

and imported into a private dataset here:
https://www.checklistbank.org/dataset/275935/imports

**Comment by yroskov** [comment]
>I have converted the excel sheet to coldp here

Thank you, @mdoering. I am not sure that the spreadsheet attached to the paper contains all data which we need (for example, synonyms and combinations are missing there). We need to contact the authors for more details after TG has confirm the interest.

**Comment by olafbanki** [comment]
Thanks Yuri! If you get a go from the COL Taxonomy Group could you also immediately make a request for either a CC-0 or a CC-BY license? The present license of the article is not workable: https://creativecommons.org/licenses/by-nc-nd/4.0/
@camiplata can assist you with some text explanation of either a CC-0 or CC-BY license.

**Comment by mdoering** [comment]
Any progress here? Is there much debate needed if we have a true gap with no species at all currently? Anything is better than that

**Comment by DaveNicolson** [comment]
I understood that complete gaps in COL should be filled with the first credible/robust data source available, and it sounds like that's this paper. If the data can be handled through Checklist Bank into COL that sounds relatively quick and easy if the author agrees to it.

Otherwise we can add it to the queue for ITIS work, so it would take time.

**Comment by mdoering** [comment]
I have digitised the list and it is accessible in CLB here:
https://www.checklistbank.org/dataset/275935/imports

941 accepted species, some infraspecies (all subspecies?) and a classification with subfamilies. There are no synonyms or vernacular names, but if the authors indeed have that somewhere these can easily be added in the next round and should not prevent us from filling this total gap.

**Comment by mdoering** [comment]
Things that can still be improved in the ColDP file: 
 - use subspecies for all infraspecific names
 - link subspecies to species properly

**Comment by camiplata** [comment]
@yroskov  should I contact the authors regarding the license ?

**Comment by yroskov** [comment]
@camiplata, oh yes, it would be nice

**Comment by dhobern** [comment]
Thanks @camiplata, that would be great. Here is what I wrote in November last year:

Dear Ana Sofia,

We are pleased to see the publication of the new paper from yourself and your colleagues on world diversity in Diplura. On behalf of the Catalogue of Life team, I am writing to ask whether you are willing to make the spreadsheet in the supplementary materials available for use as an update to this section of the Catalogue of Life Checklist. This would have several consequences, including:

    A version of your dataset would be discoverable, searchable and downloadable via a DOI on the ChecklistBank website (https://checklistbank.org/).
    Your species list would become the structure for Diplura data in GBIF and various other biodiversity data infrastructures.
    We would be able to provide feedback and reporting on the use of your data through Catalogue of Life.
    If you wish, we could also assist with getting the data into a platform that would make it easier to maintain and edit it collaboratively, or alternatively, we would be able to accept new versions of the spreadsheet with updates and corrections and show these as new versions (with a change history) in ChecklistBank.

If you are interested, I would be happy to discuss this further. The only obvious issue I can see right now is that your paper is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives licence. We can certainly provide clear attribution to you, your authors and to the paper, but the Catalogue of Life Checklist is published under a plain Creative Commons Attribution licence to ensure that these basic reference data can be used consistently anywhere by anyone. In principle, a paid environmental consultancy would be barred from using NonCommercial datasets. In the same way, we cannot handle a NoDerivatives licence, since we expect to support reuse in multiple formats and allow for users to access subsets or combinations of data from multiple species lists.

If you and your colleagues are comfortable with a Creative Commons Attribution (CC BY) licence for the supplementary spreadsheet, we can handle all the other aspects.

Many thanks for your consideration.

Best wishes,

Donald

**Comment by camiplata** [comment]
@dhobern my apologies I didn't remember you had been in contact with them.  Did they give any kind of answer?

**Comment by dhobern** [comment]
No reply at all - and I believe I sent a follow-up message as well.

Donald

-- 

*Donald Hobern* / ***@***.*** / +61 420511471
Araba Bioscan Project <https://stangeia.hobern.net/araba-bioscan-project/>
/ Pterophoroidea <https://pterophoroidea.hobern.net/> / Alucitoidea
<https://alucitoidea.hobern.net/> / BOLD Australia
<https://bold-au.hobern.net/>
ORCID: 0000-0001-6492-4016 <https://orcid.org/0000-0001-6492-4016> / Blog
<https://stangeia.hobern.net/> / iNaturalist
<https://inaturalist.ala.org.au/people/dhobern> / Flickr
<https://www.flickr.com/photos/dhobern/>/ GitHub
<https://github.com/dhobern> / Mastodon ***@***.***>

**Comment by camiplata** [comment]
I found a new email address and wrote her again, lets see if we have a response with this new communication.

**Comment by yroskov** [comment]
Than you, @camiplata!

**Comment by camiplata** [comment]
New email sent to the first author of the paper ALBERTO SENDRA

**Comment by camiplata** [comment]
We haven't got any answer yet. @DaveNicolson Could you please consider adding this to the queue for ITIS ? both the supplementary material and [the dataset](https://www.checklistbank.org/dataset/275935/about) prepared by Markus could aid this work.

**Comment by DaveNicolson** [comment]
Daniel is already working on it (adding generic authorship, double-checking and adding some synonymy, and basic geographic info). Not sure when it'll be loaded into ITIS, but he's approaching completion of the initial edits, to be followed by 2 rounds of checks by others here, and my final checks. I'd guess 1st quarter of 2025.

**Comment by DaveNicolson** [comment]
Complete lack of synonymy has been an issue, as surveying relevant literature revealed some names in use that aren't accounted for in the paper.

**Comment by camiplata** [comment]
That's great @DaveNicolson,  would it be useful for you and your team to have all the names for Diplura that we currently have from the xrelease]? the main sources are ITIS, PLAZI, PaleoBioDB and TAXREF.

**Comment by DaveNicolson** [comment]
@camiplata yes, Daniel is happy to have additional materials. Integrating everything may not be possible, given the lack of synonymy in the published list, but Daniel is doing what he can. Please email me any further info.

**Comment by yroskov** [comment]
@DaveNicolson, Dear Dave, is there any progress with Diplura's update in ITIS? Can the checklist be taken in CoL from ITIS as a global checklist or as a "global checklist in progress"? 

(Latest public version of ITIS has 163 spp vs 941 spp in this version of digitized paper https://www.checklistbank.org/dataset/275935/classification?taxonKey=x8 and 1008 spp in this version https://www.dev.checklistbank.org/dataset/307624/classification

**Comment by DaveNicolson** [comment]
We do have a working file that includes the names from that 2021 paper, thanks to your reminder I realize we need to push it to completion... So thanks for helping me get it back on track! We will also make further updates based on subsequent literature, so that it will hopefully be even more complete/current.
